.kokoro/nightly/create-versions-csv.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#input -> text file that lists versionless coordinates
+#
+#example ->
+#com.google.cloud:google-cloud-storage
+#com.google.cloud:google-cloud-vision
+
+#Output:
+#cloud_java_client_library_release_dates.csv that holds the data defined below. It has artifact_id,service_name,version, and release_date columns.
```

Comment:
The job needs to upload the data to the GCP project so that the BigQuery table is updated.

Reply:
updated comments.